v20.2/create-role.md

```diff
@@ -38,13 +38,259 @@ The `CREATE ROLE` [statement](sql-statements.html) creates SQL [roles](authoriza
+~~~
+
+~~~ sql
+root@:26257/defaultdb> SHOW ROLES;
+~~~
+
+~~~
+ username  |                options                | member_of
+-----------+---------------------------------------+------------
+admin      |                                       | {}
+can_login  | VALID UNTIL=2021-10-10 00:00:00+00:00 | {}
+no_options | NOLOGIN                               | {}
+root       |                                       | {admin}
+(4 rows)
+~~~
+
+### Prevent a role from using password authentication
+
+The following statement prevents the role from using password authentication and mandates certificate-based client authentication:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> CREATE ROLE no_password WITH PASSWORD NULL;
+~~~
+
+~~~ sql
+root@:26257/defaultdb> SHOW ROLES;
+~~~
+
+~~~
+ username  |                options                | member_of
+-----------+---------------------------------------+------------
+admin      |                                       | {}
+can_login  | VALID UNTIL=2021-10-10 00:00:00+00:00 | {}
+no_options | NOLOGIN                               | {}
+no_password| NOLOGIN                               | {}  
+root       |                                       | {admin}
+(5 rows)
+~~~
+
+### Create a role that can create other roles and manage authentication methods for the new roles
+
+~~~ sql
+root@:26257/defaultdb> CREATE ROLE can_create_role WITH CREATEROLE CREATELOGIN;
+~~~
+
+~~~ sql
+root@:26257/defaultdb> SHOW ROLES;
+~~~
+
+~~~
+   username     |                options                | member_of
+----------------+---------------------------------------+------------
+admin           |                                       | {}
+can_create_role | CREATELOGIN, CREATEROLE, NOLOGIN      | {}
+can_login       | VALID UNTIL=2021-10-10 00:00:00+00:00 | {}
+no_options      | NOLOGIN                               | {}
+no_password     | NOLOGIN                               | {}
+root            |                                       | {admin}
+(6 rows)
+~~~
+
+### Create a role that can create and rename databases
+
+~~~ sql
+root@:26257/defaultdb> CREATE ROLE can_create_db WITH CREATEDB;
+~~~
+
+~~~ sql
+root@:26257/defaultdb> SHOW ROLES;
+~~~
+
+~~~
+      username        |                options                | member_of
+----------------------+---------------------------------------+------------
+admin                 |                                       | {}
+can_create_db         | CREATEDB, NOLOGIN                     | {}
+can_create_role       | CREATELOGIN, CREATEROLE, NOLOGIN      | {}
+can_login             | VALID UNTIL=2021-10-10 00:00:00+00:00 | {}
+no_options            | NOLOGIN                               | {}
+no_password           | NOLOGIN                               | {}
+root                  |                                       | {admin}
+(7 rows)
+~~~
+
+### Create a role that can pause, resume, and cancel non-admin jobs
```

Comment:
Might be worth adding an opening sentence that links to the various `* JOBS` docs